initrd/dockerfile.go

```diff
@@ -381,14 +383,35 @@ func (initrd *dockerfile) Build(ctx context.Context) (string, error) {
 				return "", fmt.Errorf("could not create symlink: %w", err)
 			}
 		case tar.TypeLink:
-			if err := os.Link(header.Linkname, targetPath); err != nil {
-				return "", fmt.Errorf("could not create hard link: %w", err)
-			}
+			hardlinks[header.Name] = header.Linkname
+		case tar.TypeBlock:
+			log.G(ctx).
+				WithField("file", header.Name).
+				Warnf("block devices not supported, check your Dockerfile, ignoring")
```

Comment:
`Warnf` -> `Warn` (no formatting)

`block devices not supported, check your Dockerfile, ignoring` -> `ignoring block devices`